src/components/bill_card/BillCard.jsx

```diff
@@ -0,0 +1,48 @@
+import "./BillCard.css";
+const BillCard = () => {
+	return (
+		<div class="card bill-card">
+			<div class="card-body">
+				<p class="typo-label">Price Details</p>
+			</div>
+			<hr />
+			<div class="card-body">
+				<div class="bill-item">
+					<p class="typo-subtext">Price (2 Items)</p>
+					<p class="typo-subtext">Rs 3998</p>
+				</div>
+
+				<div class="bill-item">
+					<p class="typo-subtext">Discount</p>
+					<p class="typo-subtext">- Rs 1000</p>
+				</div>
+				<div class="bill-item">
+					<p class="typo-subtext">Dilevery charges</p>
+					<p class="typo-subtext">Rs 199</p>
+				</div>
+				<hr />
+				<div class="bill-item">
+					<p class="typo-subtext text-bold">Total Amount</p>
+					<p class="typo-subtext">Rs 4198</p>
+				</div>
+				<hr />
+				<div class="bill-item">
+					<p class="typo-label typo-subtext">
+						<i class="fas fa-tags"></i>
+						Use coupan
+					</p>
+					<button class="btn btn-success p-0">Apply</button>
```

Comment:
suggestion: use coupon should be an input, so we can type coupon code.

Reply:
I have not implemented that part yet, so as of now i put like that. I will change it once i implement that feature.